### PR TITLE
feat(shorebird_cli): show usage per app for current billing cycle

### PR DESCRIPTION
**Pull request by felangel**
## Description

image

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/shorebird_cli/lib/src/code_push_client_wrapper.dart [hunk 1]
```diff
@@ -528,17 +528,17 @@ aar artifact already exists, continuing...''',
...
   }
 
-  Future<List<AppUsage>> getUsage() async {
+  Future<GetUsageResponse> getUsage() async {
     final progress = logger.progress('Fetching usage');
-    final List<AppUsage>? appsUsage;
+    final GetUsageResponse? usage;
     try {
-      appsUsage = await codePushClient.getUsage();
+      usage = await codePushClient.getUsage();
```
Comment (bryanoltman) [inline packages/shorebird_cli/lib/src/code_push_client_wrapper.dart]: You can declare `usage` in the scope of this try and return below the `progress.complete()`

File: packages/shorebird_code_push_client/lib/src/version.dart [hunk 2]
```diff
@@ -1,2 +1,2 @@
 // Generated code. Do not modify.
-const packageVersion = '0.1.0+1';
+const packageVersion = '0.2.0+1';
```
Comment (bryanoltman) [inline packages/shorebird_code_push_client/lib/src/version.dart:2]: Should this be in a separate commit?

File: packages/shorebird_code_push_client/pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: shorebird_code_push_client
 description: Library which allows Dart applications to interact with the ShoreBird CodePush API
-version: 0.1.0+1
+version: 0.2.0+1
```
Comment (bryanoltman) [inline packages/shorebird_code_push_client/pubspec.yaml:3]: Same here re: a separate commit

File: packages/shorebird_code_push_client/lib/src/version.dart (hunk 2 above)
Comment (felangel) [reply packages/shorebird_code_push_client/lib/src/version.dart:2]: figured I'd make it here since this is a breaking change and I don't want to forget

**Review by bryanoltman — APPROVED** [review]
